### ISSUE TITLE
Include Apollo 5 as supported mission

### ISSUE THREAD

**Issue by ThymoNL** [dseagrav/NASSP#675]
Apollo 5 has been working for some time now but could do with some more public testing. After a period in beta it can be marked stable for NASSP 8.

**Comment by rcflyinghokie** [comment]
@indy91 this can be closed now, correct?

**Comment by indy91** [comment]
Yes, I think so. The T-60s scenario is the one that is actually supported, but it's a bit outdated. I don't have a Checklist MFD for the LM power up. I will do a test of the T-60s scenario some time and if it still works fine then I will close this issue.

**Comment by rcflyinghokie** [comment]
Functions correctly, just need to adjust so it initializes correctly (ie on the pad not in space) and connects to GSE on the pad (for cooling)
